wp-conditional-blocks.php

```diff
@@ -53,13 +53,14 @@ function () {
 
 // Load the plugin's main files.
 require_once __DIR__ . '/src/assets.php';
-require_once __DIR__ . '/src/class-wp-conditional-blocks.php';
+require_once __DIR__ . '/src/class-conditions.php';
 require_once __DIR__ . '/src/meta.php';
+require_once __DIR__ . '/src/endpoints.php';
 
 /**
  * Instantiate the plugin.
  */
 function main(): void {
-	WP_Conditional_Blocks::get_instance();
+	Conditions::get_instance();
```

Comment:
Same comment as above re: the feature class

Reply:
I'll hold off on implementing here, until I know I did it correctly for the `register_endpoints()`